src/index.css

```diff
@@ -8,4 +8,34 @@
 @font-face {
     font-family: "Montserrat";
     src: url('./fonts/Montserrat-Regular.ttf') format('truetype');
+}
+
+body {
+    margin: 0;
+}
+
+/* HEADER */
+
+nav {
```

Comment:
I'd also suggest not to target HTML elements like this if somehow possible. Rather use classes. The only exception is if you want to define styles globally as default behavior (e.g. for your default text or headings)